README.md

```diff
@@ -166,6 +166,39 @@ letsencrypt::certonly { 'foo':
   plugin        => 'dns-rfc2136',
 }
 ```
+#### dns-ovh plugin
+
+To request a certificate using the `dns-ovh` plugin, you will at a minimum
+need to pass `endpoint`, `application_key`, `application_secret` and
+`consumer_key`to the class `letsencrypt::plugin::dns_ovh`.
+Ideally keys and secret should be encrypted, eg. with eyaml if using Hiera.
+
+Plugin documentation and it's parameters can be found here:
+https://certbot-dns-ovh.readthedocs.io
+
+Note:
+
+For Debian based OS, this plugin is compatible from Debian 10
```

Comment:
Why is it not supported? If there are no packages, would it be better to set the package name to undef on those and add a check in the class that if `manage_package` is true, the package must be set?